internal/server/event_handlers.go

```diff
@@ -157,6 +158,10 @@ func (s *EventServer) handleEvent() func(w http.ResponseWriter, r *http.Request)
 				if t, ok := secret.Data["token"]; ok {
 					token = string(t)
 				}
+
+				if p, ok := secret.Data["port"]; ok {
```

Comment:
Why is the port a "secret"? The URL should contain the port if it's not 80 or 443.

Reply:
I will take the port out